lib/utils/utilities.js

```diff
@@ -61,23 +62,31 @@ function getSimulatorsFolder () {
 }
 
 function getSimulatorModelId (cli, target) {
-    target = target || '^iPhone';
+    target = new RegExp(target || '^iPhone');
 
-    // Command to fetch for a list of simulators by target. defaults to (^iPhone)
-    const cmd = `${cli} run --list --emulator${module.exports.PARAMEDIC_COMMON_CLI_ARGS} | grep "${target}" | tail -n1`;
+    const args = [
+        'run',
+        '--list',
+        '--emulator'
+    ].concat(module.exports.PARAMEDIC_COMMON_ARGS);
 
+    // Fetches all known simulators/emulators.
     logger.info('running:');
-    logger.info('    ' + cmd);
+    logger.info(`    ${cli} ${args.join(' ')}`);
 
-    // Results of the fetching command.
-    const results = exec(cmd);
+    const result = execa.sync(cli, args);
 
-    if (results.code > 0) {
+    if (result.exitCode > 0) {
         logger.error('Failed to find simulator we deployed to');
         return;
     }
 
-    return results.stdout;
+    // Return the individual target that is filtered from the known simulators/emulators based on provided target name. (default: ^iPhone)
+    return result.stdout
+        .split('\n')
+        .filter(i => i.match(target))
+        .pop()
```

Comment:
this uses the last match?

Reply:
This will return the last item from the filtered list.

Filter list for, with the target of `^iPhone`, for example:
```
iPhone-5s, 12.2
iPhone-6, 12.2
iPhone-6-Plus, 12.2
iPhone-6s, 12.2
iPhone-6s-Plus, 12.2
iPhone-7, 12.2
iPhone-7-Plus, 12.2
iPhone-8, 12.2
iPhone-8-Plus, 12.2
iPhone-SE, 12.2
iPhone-X, 12.2
iPhone-XS, 12.2
iPhone-XS-Max, 12.2
iPhone-XR, 12.2
```
`pop()` will return `iPhone-XR, 12.2`